test/modules/fetch/intercept/fetch.test.ts

```diff
@@ -329,3 +330,22 @@ it('intercepts an HTTPS PATCH request', async () => {
 
   expect(requestId).toMatch(REQUEST_ID_REGEXP)
 })
+
+it.only('support content-encoding: gzip, br', async () => {
```

Comment:
Let's be careful not to let this slip through 👀

Reply:
I think there is a lint rule for this. Do you want me to check it out?

Reply:
If this project uses eslint, then let's do that!

Reply:
We don't use eslint in this repo? 
I mean to this rule: https://www.npmjs.com/package/eslint-plugin-no-only-tests

I also see that `vitest` support this OOTB and our test should fail: https://github.com/vitest-dev/vitest/pull/669